Source: Arch Linux
Upstream: Unknown
Reason: Fixes compilation with enchant-2

--- a/gtkspell/gtkspell.c
+++ b/gtkspell/gtkspell.c
@@ -277,7 +277,7 @@
 	get_word_extents_from_mark(spell->buffer, &start, &end, spell->mark_click);
 	word = gtk_text_buffer_get_text(spell->buffer, &start, &end, FALSE);
 	
-	enchant_dict_add_to_pwl( spell->speller, word, strlen(word));
+	enchant_dict_add( spell->speller, word, strlen(word));
 
 	gtkspell_recheck_all(spell);
